src/services/expensesServices.js

```diff
@@ -0,0 +1,62 @@
+const { Expense } = require('./../models/Expense.model');
+const { Op } = require('sequelize');
+const normalize = ({ amount, category, id, note, spentAt, title, userId }) => {
+  return {
+    amount,
+    category,
+    id,
+    note,
+    spentAt,
+    title,
+    userId,
+  };
+};
+
+const getAll = async ({ categories, from, to, ...query }) => {
+  const where = { ...query };
+
+  if (categories) {
+    where.category = categories;
+  }
+
+  if (from && to) {
+    where.spentAt = { [Op.between]: [new Date(from), new Date(to)] };
+  } else if (from) {
+    where.spentAt = { [Op.gte]: [from] };
+  } else if (to) {
+    where.spentAt = { [Op.lte]: [to] };
```

Comment:
Potential Issue: The `[Op.lte]: [to]` should be `[Op.lte]: new Date(to)` to ensure that the date is properly formatted as a Date object.